packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -139,7 +140,7 @@ const BlockInspector = ( { showNoBlockSelectedMessage = true } ) => {
 						/>
 						<InspectorControls.Slot
 							group="border"
-							label={ __( 'Border' ) }
+							label={ getBorderPanelLabel( selectedBlockName ) }
```

Comment:
> I guess with the current UI arrangement of the border panel, keeping the label as border is appropriate as it is acting as field label rather than a group label.

I don't think I understand this comment; could you elaborate?

Reply:
I mean the border control (first control) doesn't have a label of its own where as Radius control has its label.
The panel heading is serving as title for the first control here.

<img width="292" alt="image" src="https://github.com/WordPress/gutenberg/assets/1935113/f862f9b9-7dc9-41aa-8ed6-017abda07407">

This is because having another label "RADIUS" makes it redundant. If we rename the panel title, we would have to introduce a label for the first control. Or we can leave the existing behaviour as is.

Reply:
Ah, makes sense; good catch! Introduced in 13cb2459f65703ff206a062b06a3319c010bce66